conference/talks.py

```diff
@@ -1,9 +1,13 @@
 from django.conf.urls import url
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
 from django.core.urlresolvers import reverse
-from django.shortcuts import get_object_or_404
+from django.http import HttpResponseForbidden
+from django.shortcuts import get_object_or_404, redirect
 from django.template.response import TemplateResponse
 
-from conference.models import Talk
+from conference.models import Conference, Talk, TALK_STATUS
+from conference.new_forms import TalkUpdateForm
```

Comment:
I'm a bit sceptical about the name. Maybe `conference.talk_forms`? (or `forms/talks.py`) WDYT?

Reply:
I like `forms/talks.py` - I've called it new_forms as I didn't want to add to the existing `forms.py`, most of which is due to be retired after ep2019.